src/reducers/combinedReducers.js

```diff
@@ -1,12 +1,18 @@
 import { combineReducers } from 'redux';
-import playersListReducer from './playersListReducer.js';
-import playerDetailsReducer from './playerDetailsReducer.js';
-import teamsListReducer from './teamsListReducer.js';
-import getGamesReducer from './getGamesReducer.js';
+import playersListReducer from './playersListReducer';
+import playerDetailsReducer from './playerDetailsReducer';
+import getGamesReducer from './getGamesReducer';
+import getSelectedTeamReducer from './getSelectedTeamReducer';
+import getTeamsReducer from './getTeamsReducer';
+import getTeamColorReducer from './getTeamColorReducer';
 
 export default combineReducers({
     players: playersListReducer,
     player: playerDetailsReducer,
-    teams: teamsListReducer,
+    // NOTE: I'll fix and rename the teams list reducer in a separate branch
```

Comment:
Change this `// NOTE:` to a `// TODO:`

Reply:
Ended up changing the reducer name so dont need the note anymore.